app/models/grantee.rb

```diff
@@ -11,43 +11,56 @@ def hses_link
     links[:html]
   end
 
-  def centers_total
-    attributes[:numberOfCenters]
+  def centers
+    current_grant&.centers
```

Comment:
as per a conversation with Laura: the "total centers" is the number of centers of the most recent grant

Reply:
Can there be more than one active grant at a time?

Reply:
she did not think so, although this is an assumption we would need to re-validate if we had more time. :/

Reply:
ok -- so should the total number of centers then be the sum of all the grant's centers? that's easier than sorting on a date etc

Reply:
I think so - it's at least a good place to start.